icons/robots.svg

```diff
@@ -0,0 +1 @@
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24"><path d="m 12,2.909 a 1.818,1.818 0 0 1 1.818,1.818 C 13.818,5.4 13.455,5.991 12.909,6.3 v 1.155 h 0.909 a 6.364,6.364 0 0 1 6.364,6.364 h 0.909 A 0.909,0.909 0 0 1 22,14.727 v 2.727 a 0.909,0.909 0 0 1 -0.909,0.909 h -0.909 v 0.909 a 1.818,1.818 0 0 1 -1.818,1.818 H 5.636 A 1.818,1.818 0 0 1 3.818,19.273 V 18.364 H 2.909 A 0.909,0.909 0 0 1 2,17.455 V 14.727 A 0.909,0.909 0 0 1 2.909,13.818 H 3.818 A 6.364,6.364 0 0 1 10.182,7.455 h 0.909 V 6.3 C 10.545,5.991 10.182,5.4 10.182,4.727 A 1.818,1.818 0 0 1 12,2.909 M 7.909,12.909 a 2.273,2.273 0 0 0 -2.273,2.273 2.273,2.273 0 0 0 2.273,2.273 2.273,2.273 0 0 0 2.273,-2.273 2.273,2.273 0 0 0 -2.273,-2.273 m 8.182,0 a 2.273,2.273 0 0 0 -2.273,2.273 2.273,2.273 0 0 0 2.273,2.273 2.273,2.273 0 0 0 2.273,-2.273 2.273,2.273 0 0 0 -2.273,-2.273 z" fill="#e53935"/></svg>
```

Comment:
Hi @4086606, I really like the icon and I just reviewed it. Would it be fine for you if we could change the color to a little paler red like `#ff5252`. Would be great if you could change that. Thank you so much!

Reply:
Done!